test/amount-test.js

```diff
@@ -1225,7 +1225,7 @@ describe('Amount', function() {
 
     it('from_json minimum IOU', function() {
       const amt = Amount.from_json('-1e-81/USD/rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh');
-      assert.strictEqual(amt.to_text(), '-1000000000000000e-96');
+      assert.strictEqual(amt.to_text(), '1000000000000000e-96');
       assert.strictEqual(amt.to_text(), Amount.min_value);
```

Comment:
This wants a '-' tacked onto the front.  Amount.min_abs_value might be clearer than Amount.min_value?